tests/cmsController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+require_once('../cmsController.php');
+require_once('../portfolioLogic.php');
+
+class StackTest extends TestCase
+{
+    //makeDropDown
+    //success
+
+    public function testDropDownSuccess ()
+    {
+        $input = [['title' => 'test']];
+        $expected = '<option value="test">test</option>';
+        $case = makeDropDown($input);
+        $this->assertEquals($case, $expected);
+    }
+
+    //failure
+    public function testDropDownFailure ()
+    {
+        $input = [['title' => 'test']];
+        $expected = '<option value="test">test</option>';
+        $case = makeDropDown($input);
+        $this->assertEquals($case, $expected);
+    }
+
+    //malformed
+    public function testDropDownMalformed ()
+    {
+        $input1 = 'title';
+        $this->expectException(TypeError::class);
+        makeDropDown($input1);
+    }
+
+//    makeImgDropDown
+//    success
+
+    public function testImgDropDownSuccess ()
```

Comment:
the fact this test and the ones below are identical to the testdropdown tests should have told you something...

Reply:
resolved